packages/front-end/components/Experiment/StartExperimentBanner.tsx

```diff
@@ -218,9 +286,53 @@ export function StartExperimentBanner({
     onStart && onStart();
   }
 
-  const allPassed = !checklist.some((c) => c.status === "error");
+  const isTaskCompleted = (currentTask: string) => {
+    const index = manualChecklistStatus.findIndex(
+      (task) => task.key === currentTask
+    );
+
+    if (index === -1 || !manualChecklistStatus[index]) {
+      return false;
+    }
+
+    return manualChecklistStatus[index].status === "complete";
+  };
+
+  const allPassed =
+    !checklist.some((c) => c.status === "error") &&
+    manualChecklist.every((task) => isTaskCompleted(task.key));
+
+  async function updateTaskStatus(checked: boolean, item: ManualChecklist) {
+    setUpdatingChecklist(true);
+    const updatedManualChecklistStatus = Array.isArray(manualChecklistStatus)
+      ? [...manualChecklistStatus]
+      : [];
+
+    const index = updatedManualChecklistStatus.findIndex(
+      (task) => task.key === item.key
+    );
+    if (index === -1) {
+      updatedManualChecklistStatus.push({
+        key: item.key,
+        status: checked ? "complete" : "incomplete",
+      });
+    } else {
+      updatedManualChecklistStatus[index] = {
+        key: item.key,
+        status: checked ? "complete" : "incomplete",
+      };
+    }
+    setManualChecklistStatus(updatedManualChecklistStatus);
+    await apiCall(`/experiment/${experiment.id}/launch-checklist`, {
```

Comment:
Wrap this in a try/catch so you can call `setUpdatingChecklist` back to false if there's an error.  Otherwise, you'll be stuck in a disabled state forever.